apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/SocialPreviewNode/SocialPreviewNode.tsx

```diff
@@ -75,6 +75,7 @@ export function SocialPreviewNode(): ReactElement {
       onSourceConnect={handleSourceConnect}
       isSourceConnected
       positionTargetHandle={false}
+      sourceHandleProps={{ isConnectableStart: showAnalytics === false }}
```

Comment:
Can you use the base node prop on line 71
```suggestion
			sourceHandle={ showAnalytics === true ? HandleVariant.Shown : HandleVariant.Disabled }
```